src/ioctls/system.rs

```diff
@@ -7,13 +7,13 @@ use std::os::raw::{c_char, c_ulong};
 use std::os::unix::io::{AsRawFd, FromRawFd, RawFd};
 
 use cap::Cap;
-use ioctls::vec_with_array_field;
 use ioctls::vm::{new_vmfd, VmFd};
-#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-use ioctls::CpuId;
 use ioctls::Result;
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+use ioctls::{CpuId, MsrList};
```

Comment:
I believe these two should also be exported as pub in src/lib.rs. Otherwise they'll not be available to external crates.